lib/ansible/modules/packaging/os/apt.py

```diff
@@ -469,6 +470,19 @@ def parse_diff(output):
     return {'prepared': '\n'.join(diff[diff_start:diff_end])}
 
 
+def mark_installed_manually(m, packages):
+    apt_mark_cmd_path = m.get_bin_path("apt-mark", required=True)
+    cmd = "%s manual %s" % (apt_mark_cmd_path, ' '.join(packages))
+    rc, out, err = m.run_command(cmd)
+
+    if APT_MARK_INVALID_OP in err:
+        cmd = "%s unmarkauto %s" % (apt_mark_cmd_path, ' '.join(packages))
```

Comment:
IMHO this should be `apt-get/aptitude unmarkauto` instead of `apt-mark unmarkauto`

Reply:
Both `apt-get` and `apt-mark` works in my testing. Is that because `apt-mark` wasn't available in older versions?

Reply:
Indeed. However I can not say what exactly should be the fallback here. I didn't find a "recent" but outdated debian version not having apt-mark with manual|auto (debian7, ubuntu12.04). apt-mark probably has a fallback to apt-get itself. Works for me. Never mind.